### ISSUE TITLE
My full conda list which I got it to work (CPU)

### ISSUE THREAD

**Issue by anttiryt** [Riccorl/Super-SloMo-tf2#13]
Name                    Version                   Build  Channel
    _libgcc_mutex             0.1                        main  
    _openmp_mutex             5.1                       1_gnu  
    _tflow_select             2.3.0                       mkl  
    absl-py                   1.3.0            py38h06a4308_0  
    aiohttp                   3.8.3            py38h5eee18b_0  
    aiosignal                 1.2.0              pyhd3eb1b0_0  
    appdirs                   1.4.4              pyhd3eb1b0_0  
    astor                     0.8.1            py38h06a4308_0  
    astunparse                1.6.3                      py_0  
    async-timeout             4.0.2            py38h06a4308_0  
    attrs                     22.1.0           py38h06a4308_0  
    black                     19.10b0                    py_0  
    blas                      1.0                         mkl  
    blinker                   1.4              py38h06a4308_0  
    brotlipy                  0.7.0           py38h27cfd23_1003  
    c-ares                    1.19.0               h5eee18b_0  
    ca-certificates           2023.01.10           h06a4308_0  
    cachetools                4.2.2              pyhd3eb1b0_0  
    certifi                   2022.12.7        py38h06a4308_0  
    cffi                      1.15.1           py38h5eee18b_3  
    charset-normalizer        2.0.4              pyhd3eb1b0_0  
    click                     8.0.4            py38h06a4308_0  
    cryptography              39.0.1           py38h9ce1e76_0  
    flit-core                 3.6.0              pyhd3eb1b0_0  
    frozenlist                1.3.3            py38h5eee18b_0  
    gast                      0.4.0              pyhd3eb1b0_0  
    google-auth               2.6.0              pyhd3eb1b0_0  
    google-auth-oauthlib      0.4.4              pyhd3eb1b0_0  
    google-pasta              0.2.0              pyhd3eb1b0_0  
    grpcio                    1.42.0           py38hce63b2e_0  
    h5py                      2.10.0           py38hd6299e0_1  
    hdf5                      1.10.6               hb1b8bf9_0  
    idna                      3.4              py38h06a4308_0  
    importlib-metadata        6.0.0            py38h06a4308_0  
    intel-openmp              2023.0.0         h9e868ea_25371  
    keras                     2.12.0                   pypi_0    pypi
    keras-preprocessing       1.1.2              pyhd3eb1b0_0  
    ld_impl_linux-64          2.38                 h1181459_1  
    libffi                    3.4.2                h6a678d5_6  
    libgcc-ng                 11.2.0               h1234567_1  
    libgfortran-ng            7.5.0               ha8ba4b0_17  
    libgfortran4              7.5.0               ha8ba4b0_17  
    libgomp                   11.2.0               h1234567_1  
    libprotobuf               3.20.3               he621ea3_0  
    libstdcxx-ng              11.2.0               h1234567_1  
    markdown                  3.4.1            py38h06a4308_0  
    markupsafe                2.1.1            py38h7f8727e_0  
    mkl                       2020.2                      256  
    mkl-service               2.3.0            py38he904b0f_0  
    mkl_fft                   1.3.0            py38h54f3939_0  
    mkl_random                1.1.1            py38h0573a6f_0  
    multidict                 6.0.2            py38h5eee18b_0  
    mypy_extensions           0.4.3            py38h06a4308_1  
    ncurses                   6.4                  h6a678d5_0  
    numpy                     1.18.5           py38ha1c710e_0  
    numpy-base                1.18.5           py38hde5b4d6_0  
    oauthlib                  3.2.1            py38h06a4308_0  
    opencv-python             4.7.0.72                 pypi_0    pypi
    openssl                   1.1.1t               h7f8727e_0  
    opt_einsum                3.3.0              pyhd3eb1b0_1  
    packaging                 23.0                     pypi_0    pypi
    pathspec                  0.10.3           py38h06a4308_0  
    pip                       23.0.1           py38h06a4308_0  
    protobuf                  3.20.3           py38h6a678d5_0  
    pyasn1                    0.4.8              pyhd3eb1b0_0  
    pyasn1-modules            0.2.8                      py_0  
    pycparser                 2.21               pyhd3eb1b0_0  
    pyjwt                     2.4.0            py38h06a4308_0  
    pyopenssl                 23.0.0           py38h06a4308_0  
    pysocks                   1.7.1            py38h06a4308_0  
    python                    3.8.16               h7a1cb2a_3  
    python-flatbuffers        2.0                pyhd3eb1b0_0  
    readline                  8.2                  h5eee18b_0  
    regex                     2022.7.9         py38h5eee18b_0  
    requests                  2.28.1           py38h06a4308_1  
    requests-oauthlib         1.3.0                      py_0  
    rsa                       4.7.2              pyhd3eb1b0_1  
    scipy                     1.6.2            py38h91f5cce_0  
    setuptools                65.6.3           py38h06a4308_0  
    six                       1.16.0             pyhd3eb1b0_1  
    sqlite                    3.41.1               h5eee18b_0  
    tensorboard               2.11.0           py38h06a4308_0  
    tensorboard-data-server   0.6.1            py38h52d8a92_0  
    tensorboard-plugin-wit    1.8.1            py38h06a4308_0  
    tensorflow                2.4.1           mkl_py38hb2083e0_0  
    tensorflow-addons         0.10.0                   pypi_0    pypi
    tensorflow-base           2.4.1           mkl_py38h43e0292_0  
    tensorflow-estimator      2.4.1              pyheb71bc4_0  
    termcolor                 2.1.0            py38h06a4308_0  
    tk                        8.6.12               h1ccaba5_0  
    toml                      0.10.2             pyhd3eb1b0_0  
    tqdm                      4.64.1           py38h06a4308_0  
    typed-ast                 1.4.3            py38h7f8727e_1  
    typeguard                 3.0.1                    pypi_0    pypi
    typing_extensions         4.4.0            py38h06a4308_0  
    urllib3                   1.26.14          py38h06a4308_0  
    werkzeug                  2.2.2            py38h06a4308_0  
    wheel                     0.38.4           py38h06a4308_0  
    wrapt                     1.14.1           py38h5eee18b_0  
    xz                        5.2.10               h5eee18b_1  
    yarl                      1.8.1            py38h5eee18b_0  
    zipp                      3.11.0           py38h06a4308_0  
    zlib                      1.2.13               h5eee18b_0  
    
   
   
   Numpy has upgrade issues, I always use 1.18.4/5. 
   
   Opencv-python is difficult.
